pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -481,17 +489,17 @@ Evaluation of a local-variable form, such as @racket[(let ([x (+ 1
 that replaces every instance of @racket[x] in @racket[_expr].
 
 @;------------------------------------------------------------------------
-@section{Variables and Locations}
+@section[#:tag "vars-and-locs"]{Variables and Locations}
 
 A @deftech{variable} is a placeholder for a @tech{value}, and
 expressions in an initial program refer to @tech{variables}. A
-@deftech{top-level variable} is both a @tech{variable} and a
+@tech{top-level variable} is both a @tech{variable} and a
 @tech{location}. Any other @tech{variable} is always replaced by a
-@tech{location} at run-time, so that evaluation of expressions
+@tech{location} at run-time; thus, evaluation of expressions
 involves only @tech{locations}. A single @deftech{local variable}
-(i.e., a non-top-level, non-module-level @tech{variable}), such as a
-procedure argument, can correspond to different @tech{locations}
-through different instantiations.
+(i.e., a non-top-level, non-module-level @tech{variable}), such as an
+argument parameter variable, can correspond to different @tech{locations}
```

Comment:
Same as above, I think "procedure argument" is better.

Maybe its worth adding a `deftech` for this. For now, my choice would be "procedure-argument variable".

Reply:
The trouble with "procedure argument variable" is that it clearly connotes a procedure call argument that is a variable, which is not what we're talking about here. I think that "procedure parameter variable" is clearly not referring to a parameter.